cloudwatch-logs-s3-forwarder/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "name_prefix" {
+  description = "A prefix that will be added to resources so that they are unique"
+}
+
+variable "filter_patterns" {
+  description = "List of CloudWatch filter patterns"
+  default     = []
+  type        = "list"
+}
+
+variable "log_group_names" {
+  description = "List of Cloudwatch log group names"
+  default     = []
+  type        = "list"
+}
+
+variable "lambda_handler" {
```

Comment:
If you are providing the zip file, isn't the handler predetermined by your packaging of said zip-file? In that case I think this variable should not be exposed.

Reply:
I agree, or at least provide a default value with the handler name in your function.

Reply:
Agree. The component should be tightly coupled to the specific lambda function at It's a design decision we have to make. We have two options 

1)  This module provisions Cloud Watch subscriptions to a set of log groups - and targets matching log events to a generic lambda function. The lambda function can be any lambda function - or the one we provide as an open source software component.  

2) This module is tightly coupled to the specific lambda function in this repo; https://github.com/TeliaSoneraNorge/cloudwatch-subscriptions-s3-writer  - If it's too generic, it's not useful.